mcp/health-tool-mcp/server.py

```
@@ -0,0 +1,1262 @@
+    except Exception:
+        pass
+    return None
+
+def _check_bal_at_startup() -> None:
+    """Log Ballerina version at startup for diagnostics."""
+    try:
+        bal = find_bal_executable()
+        if not bal:
+            print("[WARN] Ballerina CLI not found on PATH. Tools will fail until installed.", flush=True)
+            return
+        result = subprocess.run(
```

Ruff (0.15.7)

[error] `subprocess` call: check for execution of untrusted input

(S603)